### ISSUE TITLE
alt text for image on website describing web fundamentals is broken

### ISSUE THREAD

**Issue by zersiax** [google/WebFundamentals#3978]
The alt text for the image near the h2 about web push notifications does not have a correct alt tag, the text to describe the image currently contains the string "Alt text".  This is a very basic accessibility problem, even lower than a low hanging fruit in my opinion.
Ironically, the rest of the webpage is pretty good a11y-wise, except for the last two headings which have the wrong heading level. Not sure how that happened :)

**Comment by Gornstats** [comment]
Hi Florian, the alt text was fixed in #3979
